### PR TITLE
(multiple) fixed formatting

### PR DESCRIPTION
**Pull request by Duke-M-commits**
thx @MakinDay

### CODE REVIEW

File: resource/closecaption_russian.txt [hunk 1]
```diff
@@ -290,7 +290,7 @@
 "[english]bastille.Use2"		"<clr:205,127,50><B>Bastille:<B> Good thinking."
 "bastille.Use2"		"<clr:205,127,50><B>Бастиль:<B> Отлично мыслишь."
 
-"[english]bastille.Use3"		"<clr:205,127,50><B>Bastille:<B> Immediatement."
+"[english]bastille.Use3"		"<clr:205,127,50><B>Bastille:<B> <I>Immediatement."
 "bastille.Use3"		"<clr:205,127,50><B>Бастиль:<B> Иммедицимо."
```
Comment (Blueberryy) [inline resource/closecaption_russian.txt:294]: You can add <I> for Russian too.

Comment (Duke-M-commits) [reply resource/closecaption_russian.txt:294]: Sure thing. With or without the acute? (english doesn't use it, but the original french version comes with it)
<B>Бастиль:<B> <I>Immédiatement.
or
<B>Бастиль:<B> <I>Immediatement.

Comment (Blueberryy) [reply resource/closecaption_russian.txt]: well just leave it in russian but with <I> at the start

Comment (Duke-M-commits) [reply resource/closecaption_russian.txt]: The <I> is meant as a marker non-native words. So combining <I> and Иммедицимо defeats the purpose.

Comment (Blueberryy) [reply resource/closecaption_russian.txt]: alright just forget about it then. i will do something about it later

**Pull request merged**